Reference/conformance/conformance.pb.swift

```diff
@@ -163,7 +163,13 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message {
   /// other serializer methods are defined in the SwiftProtobuf library. See the
   /// `Message` and `Message+*Additions` files.
   func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V) throws {
-    try self.payload?.traverse(visitor: &visitor)
+    switch self.payload {
```

Comment:
These could get tweaked to be:

```swift
if let o = self.payload {
  switch o {
  ...
```
And then the `default: break` would only be needed when the oneof has to get split up because of intermixed other fields.